### DISCUSSION TITLE
Is possible to make a relationship that filter selected content from another collection?

### DISCUSSION THREAD

**Discussion by Thremulant** [strapi/strapi#7058] [Legacy Topics]
Hi, 
The use case I'm working on is a soccer tournament, where the users can create their own team. I created a collection for teams and I'm using the default user's collection, but at this point, every user has access to all the other teams in that collection. 

Do you guys know if it's possible to make a relationship that filter the content created by that user? 

Thanks!

**Comment by patrickserrano** [comment]
I'm interested to hear if there's a way to do this. I think it might inform one of the use cases I'm looking at for building rules.

**Comment by yusufisl** [comment]
I think you need this
[https://strapi.io/documentation/v3.x/guides/is-owner.html#create-is-owner-policy](https://strapi.io/documentation/v3.x/guides/is-owner.html#create-is-owner-policy)

**Reply by lauriejim** [reply]
This will definitely help.
And a quick win could be that for fetch `localhost:1337/teams?members.id_contains=1`
`1` is your current user ID
